pkgs/tools/package-management/cargo-update/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, defaultCrateOverrides, fetchFromGitHub, cmake, libssh2, libgit2, openssl, zlib }:
+
+((import ./cargo-update.nix).cargo_update {}).override {
```

Comment:
As mentioned in the PR description: I need to use `carnix` because `rustPlatform.buildRustPackage` depends on having an up-to-date `Cargo.lock` at the root of the `src`, but in the repo is no `Cargo.lock` checked in.